src/main/java/com/yeahx4/jamza/map/Map.java

```diff
@@ -0,0 +1,85 @@
+package com.yeahx4.jamza.map;
+
+import com.yeahx4.jamza.champion.Player;
+import com.yeahx4.jamza.util.Console;
+
+import java.io.Serializable;
+import java.util.LinkedHashMap;
+
+/**
+ * Map super class
+ *
+ * @author hellun205
+ * @since 1.0
+ */
+public abstract class Map implements Serializable {
+    public final String name;
+    public final String description;
+    public final boolean combatable;
+
+    /**
+     * The choices to perform on this map.
+     * {@link #onPerform(String)} must have an actions on the options.
+     */
+    public final LinkedHashMap<String, String> optionsToPerform;
+
+    /**
+     * @param name the name of the map
+     * @param description description of the map
+     * @param combatable combat availability
+     * @param optionsToPerform The choices to perform on this map.
+     *                         {@link #onPerform(String)} must have an actions on the options.
+     */
+    protected Map(String name, String description, boolean combatable, LinkedHashMap<String, String> optionsToPerform) {
+        this.name = name;
+        this.description = description;
+        this.combatable = combatable;
+        this.optionsToPerform = optionsToPerform;
+    }
+
+    /**
+     * Invoked when located on this map.
+     * The choices to perform on this map are output.
+     */
+    public void whenLocated() {
+        boolean done = false;
+        do {
+            String input = Console.selectc(String.format("&y%s&n에서 무엇을 하시겠습니까?", name), new LinkedHashMap<>() {
+                {
+                    put("info", "맵 정보 보기");
+                    putAll(optionsToPerform);
+                    put("end", "게임 종료");
+                }
+            });
+
+            switch (input) {
+                case "info":
+                    Console.printlnc(String.format("&y%s&n\n%s", name, description));
+                    Console.readLine();
+                    break;
+                case "end":
+                    boolean exitQuestion = Console.select("게임을 종료하시겠습니까?", new LinkedHashMap<Boolean, String>() {
```

Comment:
Game wont be ended just for exiting load method and game cycle must not be handled in Map class